EnglishTranslation-master/ITEM.tsv

```diff
@@ -9268,11 +9268,11 @@ ITEM_20151224_009267	Bead Amulet
 ITEM_20151224_009268	Cross Medal Amulet
 ITEM_20151224_009269	Gem Amulet
 ITEM_20151224_009270	Dilgele Seeds
-ITEM_20151224_009271	You can plant these seeds to grow produce. {nl}They can be used only within your guild's hideout.
+ITEM_20151224_009271	You can plant these seeds to grow crops.{nl}Can only be used in a Guild Hangout.
 ITEM_20151224_009272	Gyslotis Seeds
 ITEM_20151224_009273	Raudonelis
 ITEM_20151224_009274	Guinea Pig Egg
-ITEM_20151224_009275	You can grow a companion from this egg. {nl}It can be used only within your guild's hangout.
+ITEM_20151224_009275	You can grow a companion from this egg. {nl}It can only be used in a Guild Hangout.
```

Comment:
Wouldn't it be better if both lines 9271 and 9275 had the same sentence in the end?